thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -3,6 +3,7 @@ import { filterObject, filterPrefix, toBaselineUrn, toCurrentUrn } from 'thirdey
 import EVENT_TABLE_COLUMNS from 'thirdeye-frontend/mocks/eventTableColumns';
 import config from 'thirdeye-frontend/mocks/filterBarConfig';
 import CryptoJS from 'cryptojs';
+const { setProperties } = Ember;
```

Comment:
since Ember 2.16, the best practice is to import modules instead of the whole ember library.
```
  import { computed, get, set, setProperties } from '@ember/object'
```
#### Official documentation: 
https://www.emberjs.com/blog/2017/10/11/ember-2-16-released.html